pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -301,3 +301,102 @@ func (p *staticPolicy) guaranteedCPUs(pod *v1.Pod, container *v1.Container) int
 	// https://golang.org/ref/spec#Numeric_types
 	return int(cpuQuantity.Value())
 }
+
+func (p *staticPolicy) GetTopologyHints(s state.State, pod v1.Pod, container v1.Container) map[string][]topologymanager.TopologyHint {
+	// If there are no CPU resources requested for this container, we do not
+	// generate any topology hints.
+	if _, ok := container.Resources.Requests[v1.ResourceCPU]; !ok {
+		return nil
+	}
+
+	// Otherwise, attempt to generate TopologyHints for the CPUManager.
+	// TODO: Generalize this so that its applicable to future CPUManager polices.
```

Comment:
Probably dont need this TODO if we go this route

Reply:
You're right. I tried to come up with a simplified interface for policy implementers but I'm not sure if it's possible. Even enumerating the possibilities depends on the CPU management policy in deep ways. Removing.

Reply:
done